src/fsdocs-tool/Options.fs

```diff
@@ -38,3 +38,131 @@ module Common =
+        member this.docs_content = Path.Combine(this.docs, "content") |> Path.GetFullPath
+        member this.docs_content_img = Path.Combine(this.docs_content, "img") |> Path.GetFullPath
+
+        // specific files in this folder structure that might need special treatment instead of just copy pasting
+        member this.template_html = Path.Combine(this.docs, "_template.html") |> Path.GetFullPath
+        member this.template_ipynb = Path.Combine(this.docs, "_template.ipynb") |> Path.GetFullPath
+        member this.template_tex = Path.Combine(this.docs, "_template.tex") |> Path.GetFullPath
+        member this.dockerfile = Path.Combine(this.docs, "Dockerfile") |> Path.GetFullPath
+        member this.nuget_config = Path.Combine(this.docs, "Nuget.config") |> Path.GetFullPath
+
+        // specific files for the init command
+        member this.logo_template = Path.Combine(this.docs_templates_init, ".logo.png") |> Path.GetFullPath
+
+        member this.index_md_template =
+            Path.Combine(this.docs_templates_init, ".index_md_template.md")
+            |> Path.GetFullPath
+
+        member this.literate_sample_template =
+            Path.Combine(this.docs_templates_init, ".literate_sample_template.fsx")
+            |> Path.GetFullPath
+
+        /// <summary>
+        /// returns true if all special files and folders of this location exist.
+        /// </summary>
+        member this.Exist() =
+            try
+                Directory.Exists(this.docs)
+                && Directory.Exists(this.docs_templates)
+                && Directory.Exists(this.docs_templates_init)
+                && Directory.Exists(this.docs_img)
+                && Directory.Exists(this.docs_content)
+                && Directory.Exists(this.docs_content_img)
+                && File.Exists(this.template_html)
+                && File.Exists(this.template_ipynb)
+                && File.Exists(this.template_tex)
+                && File.Exists(this.dockerfile)
+                && File.Exists(this.nuget_config)
+                && File.Exists(this.logo_template)
+                && File.Exists(this.index_md_template)
+                && File.Exists(this.literate_sample_template)
+            with _ ->
+                false
+
+    /// <summary>
+    /// a set of default locations in the nuget package created for fsdocs-tool.
+    /// these files are to be used in 2 scenarios assuming the tool is invoked via cli:
+    ///
+    /// - as default styles when running watch or build when there are no user equivalents present and `nodefaultcontent` is not set to true
+    ///
+    /// - as content of the output of the `init` command to initialize a default docs folder structure.
+    ///
+    /// Note that the path of these files will always be combined with the given `assemblyPath` because the cli tool will query it's own path on runtime via reflection.
+    /// </summary>
+    type InPackageLocations(relAssemblyPath) =
```

Comment:
My exact thoughts initially, but the project does pack the files slightly different into the nuget package than the actual paths in the repo, see here for example:

https://github.com/fsprojects/FSharp.Formatting/blob/7707ea0be16eb98dbe0fe3b94e4dac0e3ddcc811/src/fsdocs-tool/fsdocs-tool.fsproj#L30-L32

a "templates" folder does not exist in the repo, but will contain the template files in the nuget package. I guess the reason here is that in the repo, we need an intact `docs` folder. It just felt wrong putting this on the same class when it could contain paths that do not exist.